src/main/resources/application-local.yaml.sample

```
@@ -25,6 +25,10 @@ spring:
 application:
   database-initializer:
     run-on-startup: true
+  feature-flags:
+    hide-manifest: # Map of status codes to boolean values. Default is false.
+      'PASSPORT_ISSUED_SHIPPING_CANADA_POST': true
+      'PASSPORT_ISSUED_SHIPPING_FEDEX': false
```

I don't think this needs to be a map..

```suggestion
  feature-flags:
    hidden-manifests:
      - PASSPORT_ISSUED_SHIPPING_CANADA_POST
```

or

```suggestion
  feature-flags:
    hidden-manifests: [PASSPORT_ISSUED_SHIPPING_CANADA_POST]
```